components/cluster.py

```diff
@@ -116,74 +118,67 @@ def build_cluster_component(mindate, maxdate, session_id, idd="1"):
     id_component = "date-pick-{}".format(idd)
     date_component = date_picker.get_component(
         mindate, maxdate, None, id_component)
+    threshold_component = cluster_config.get_component(idd)
     return html.Div([
         date_component,
+        html.Div(children="Type threshold:"),
```

Comment:
These might make more sense if they were added to the `cluster_config` component. Adding the class `config-label` to the "Type threshold" label will keep its styling consistent with the rest of the views